charts/fybrik/files/taxonomy/base_taxonomy.json

```diff
@@ -0,0 +1,190 @@
+{
+  "title": "taxonomy.json",
+  "definitions": {
+    "Action": {
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "$ref": "#/definitions/ActionName"
+        }
+      },
+      "additionalProperties": true
+    },
+    "ActionName": {
+      "type": "string"
+    },
+    "AppInfo": {
+      "type": "object",
+      "additionalProperties": true
+    },
+    "AssetID": {
+      "description": "Asset ID of the asset to be queried in the catalog",
```

Comment:
In the read flow the asset is being queried, but in the write flow it is being written

Reply:
Changed the description